backend/Recodo.DAL/Entities/EnumReaction.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recodo.DAL.Entities
+{
+    public enum Reaction
```

Comment:
I think you can move this and other enums to the Common project. It can be reused.